src/main/java/com/group1/stagesWs/controller/UserController.java

```diff
@@ -73,6 +66,12 @@ public UserController(UserService service) {
         return new ResponseEntity<>(service.getAllEtudiants(), HttpStatus.OK);
     }
 
+    @GetMapping(path = "/etudiants/{idSuperviseur}")
```

Comment:
I might suggest using a different path for this. The path should be indicative of what you're returning
```
/user/etudiants/{id}
```
To me the above link should return an etudiant, not the list of etudiants under a superviseur.

For example, a more descriptive path might be
```
/user/superviseur/{id}/etudiants
```
or something like that.